src/main/java/org/gridsuite/filter/server/FilterController.java

```diff
@@ -97,6 +118,15 @@ public ResponseEntity<Void> changeFilter(@PathVariable UUID id, @RequestBody Abs
         }
     }
 
+    @PutMapping(value = "/filters/batch", consumes = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Modify filters in batch")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "Filters has been successfully modified")})
+    public ResponseEntity<List<AbstractFilter>> changeFilters(@RequestBody Map<UUID, AbstractFilter> filtersToModifyMap) {
```

Comment:
How does it change the filters ? Can you be more specific in the function name and/or @Operation(summary ?

Reply:
DONE in https://github.com/gridsuite/filter-server/pull/111/commits/f551b4c435a76745c6a987f1830a5c6f3f535c20